docs/specification/draft/server/profiles.mdx

```diff
@@ -0,0 +1,137 @@
+---
+title: Profiles
+---
+
+<div id="enable-section-numbers" />
+
+<Info>**Protocol Revision**: draft</Info>
+
+The Model Context Protocol enables the definition of a collection of behaviors and properties as Profiles. Standard Profiles are available through the Model Context Protocol, and anyone may define custom Profiles. MCP Servers can declare the profiles they support. MCP Clients can discover which profiles an MCP Server supports, request the use of specific profiles, and negotiate a mutually agreed profile to use within a session with the MCP Server.
+
+## Profile Specification
+Each Profile has a specification document that describes the profile in detail. The profile specification is a document written using best practices prescribed in [RFC 2119](https://datatracker.ietf.org/doc/html/rfc2119) that unambiguously specifies the characteristics of a profile.
+
+### Profile URL
+
+A profile specification is available at the URL that is used to uniquely identify the profile. For example:
+
+```
+https://mycompany.com/mycustomprofile-1.0
+```
+
+OR
+
+```
+https://modelcontextprotocol.io/specification/2025-06-18/server/standard-profiles/minimum-authorization-profile
+```
+
+### Standard and Custom Profiles
+- Standard profiles are governed by the Model Context Protocol and stored in the "Standard Profiles" folder in the Server section of this specification. Standard profiles always have the profile URL prefix:
+  ```
+  https://modelcontextprotocol.io/specification/<version>/server/standard-profiles/
+  ```
+- Any profiles available outside of the standard profiles location are called Custom Profiles.
+- MCP Servers may simultaneously support a mix of standard and custom profiles.
+
+### Profile Modification and Versioning
+Once made available, Profiles MUST NOT be changed. In order to make changes to a Profile, the author MUST create a new version of the Profile, with its own Profile URL.
+
+## Publishing Profiles
+An MCP Server makes it possible for Clients to discover the profiles it supports as follows:
+
+### ProfileSpec Object
```

Comment:
We're jumping WAY into the implementation details here. I am not even sure how profiles are used - a TypeScript example is way too early. A TypeScript example is also a _concrete_ implementation rather than a normative spec example.